README.md

```diff
@@ -6,47 +6,15 @@ In this codelab, you will follow a step-by-step guide to experiment with state-o
 
 # User Guide
 
-[![Open in Azure Cloud Shell](https://img.shields.io/badge/Azure%20Cloud%20Shell-Open-blue?logo=microsoft-azure)](https://ms.portal.azure.com/#cloudshell/)
-
-### Persist Cloud Shell data to avoid accidental closure (Optional - Skip to [Setup Tests](#setup-tests))
-
-#### Reset User Settings
-
-* If this is your first time using Cloud Shell, skip to [here](#set-up-storage-account)
-* If you already set up Cloud Shell without Storage Account, reset your user settings by choosing **Settings** in the top panel. Next, choose **Reset User Settings**
-
-<img src="docs/img/reset.png" alt="Reset user settings" width="960" height="600">
-
-* Click **Reset** to confirm
-
-<img src="docs/img/reset_confirm.png" alt="Confirm reset" width="960" height="600">
-
-#### Set Up Storage Account
-
-* Choose **Bash** as shell mode
+* This lab can be run on MacOS or Ubuntu
+* If you have neither, you can leverage Azure Cloud Shell 
```

Comment:
Bash Shell is required, Azure Cloud Shell with storage account is recommended, alternatively can also run MacOS or Ubuntu locally

Reply:
updated